packages/devtools_app/lib/src/screens/inspector/inspector_tree_controller.dart

```diff
@@ -700,27 +701,27 @@ extension RemoteDiagnosticsNodeExtension on RemoteDiagnosticsNode {
 abstract class InspectorControllerClient {
   void onChanged();
 
-  void scrollToRect(Rect rect);
+  void scrollToRect(Rect? rect);
 
   void requestFocus();
 }
 
 class InspectorTree extends StatefulWidget {
   const InspectorTree({
-    Key key,
-    @required this.controller,
-    @required this.debuggerController,
+    Key? key,
+    required this.controller,
+    required this.debuggerController,
     this.inspectorTreeController,
     this.isSummaryTree = false,
     this.widgetErrors,
   })  : assert(isSummaryTree == (inspectorTreeController == null)),
         super(key: key);
 
-  final InspectorTreeController controller;
-  final InspectorTreeController inspectorTreeController;
+  final InspectorTreeController? controller;
```

Comment:
Assigned to nullable value in InspectorScreen.build.

Reply:
Add a TODO here. something is wrong. One of controller and inspectorTreeController isn't really used.

Reply:
Also add a todo to follow up making this non-nullable. I think null checks may need to be added to InspectorScreen. Would be cleaner if null values aren't allowed to flow in here. Seems like all the usages are assuming this is non-nullable.